test/Microsoft.ML.Tests/LearningPipelineTests.cs

```diff
@@ -137,36 +137,6 @@ public void BooleanLabelPipeline()
             var model = pipeline.Train<Data, Prediction>();
         }
 
-        public class NullableBooleanLabelData
-        {
-            [ColumnName("Features")]
-            [VectorType(2)]
-            public float[] Features;
-
-            [ColumnName("Label")]
-            public bool? Label;
-        }
-
-        [Fact]
-        public void NullableBooleanLabelPipeline()
```

Comment:
I thought we decided we wanted to support `bool?`?

Reply:
We appear to have tests that use it - https://github.com/dotnet/machinelearning/pull/692/files#diff-a18fb7e769859c74ab2d303040626b57.

Reply:
Nope, Tom, didn't think there is a compelling reason to support bool? But if you can think of something then let me know. There were two places where bool? was being used in the codebase and they were both in prediction label from score, i.e if score was null then assign predicted label as NA but we defaulted to false for missing and you can see the change in test results. Tom is ok with this.

---
In reply to: [211434928](https://github.com/dotnet/machinelearning/pull/692#discussion_r211434928) [](ancestors = 211434928)